### PR TITLE
Demonstrate JS testing

### LINKED ISSUE #62: Demonstrate JS/jQuery unit testing
While our overall test coverage is good, JS is only covered by system tests. It would be good to at least have a handle on how we would write unit tests for JS.

jQuery is a little awkward to test because it's often going to depend on particular HTML structures that come from the ERB, so we would need to recreate these structures in the JS test, and these will need to be kept in sync.

Jest is a popular JS testing framework: [Their docs](https://jestjs.io/docs/tutorial-jquery) provide an example of testing jQuery code.

### PR DESCRIPTION
**Pull request by mccalluc**
I had started this before the meeting where we closed
- #62

We don't really need it for anything right now: I think it would be fine to close this unmerged... but I did learn something by implementing it, and it might be useful in the future. Your call.

### CODE REVIEW

**Comment by mccalluc** [comment]
Ruby test fails because Geckodriver installation doesn't work... I don't think changes in this PR would have touched that. Will rerun.

**Comment by mccalluc** [comment]
I had started this before the meeting where we closed
- #62

We don't really need it for anything right now: I think it would be fine to close this unmerged... but I did learn something by implementing it, and it might be useful in the future. Your call.

File: app/javascript/entrypoints/helper.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line import/prefer-default-export
+export function setTargetHtml(event, html) {
```
Comment (jrgriffiniii) [inline app/javascript/entrypoints/helper.js:2]: 👍 I am in no way opposed to this or critical, but was this separated in order to decouple from jQuery? I can follow this pattern if it is preferred, but I just wish to be consistent should I implement JS elsewhere.

File: app/javascript/tests/helper.test.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+describe('helper', () => {
+  it('has a window', () => {
+    expect(typeof window).toBe('object');
+  });
+
+  it('can test jQuery', () => {
+    const dom = new JSDOM('<!DOCTYPE html><h1>Hello world</h1></html>');
+    const h1 = dom.window.document.querySelector('h1');
+    setTargetHtml({ target: h1 }, 'Goodbye');
+    expect(h1.innerHTML).toEqual('Goodbye');
```
Comment (jrgriffiniii) [inline app/javascript/tests/helper.test.js:19]: 🎉

**Review by jrgriffiniii — APPROVED** [review]
Thank you very much for this! I did have one question, however I can raise this during the check-in meeting.

File: app/javascript/entrypoints/helper.js (hunk 1 above)
Comment (mccalluc) [reply app/javascript/entrypoints/helper.js:2]: Fuller discussion in standup. For reference, it's not about decoupling jQuery, it's about moving testable functions/classes out of application.js and into modules that just export with no side effects.

**Pull request merged**